lib/record_loader/pipeline_loader.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+# This file was automatically generated via `rails g record_loader`
+
+# RecordLoader handles automatic population and updating of database records
+# across different environments
+# @see https://rubydoc.info/github/sanger/record_loader/
+module RecordLoader
+  # Creates the specified plate types if they are not present
```

Comment:
Again comments don't seem to reflect the actual. Some more documentation would be good as it looks quite complex.

Reply:
Correcting